app/models/nic/base.rb

```diff
@@ -22,7 +22,7 @@ class Base < ActiveRecord::Base
     validates :mac, :uniqueness => {:scope => :virtual},
               :if => Proc.new { |nic| nic.managed? && nic.host && nic.host.managed? && !nic.host.compute? && !nic.virtual? }, :allow_blank => true
     validates :mac, :presence => true,
-              :if => Proc.new { |nic| nic.managed? && nic.host && nic.host.managed? && !nic.host.compute? &&!nic.virtual? }
+              :if => Proc.new { |nic| nic.managed? && nic.host && nic.host.managed? && !nic.host.compute? && !nic.virtual? && SETTINGS[:unattended] }
```

Comment:
maybe we should also put `&& SETTINGS[:unattended]` to orchestration modules into methods like `dhcp?`, `dns?`, ...?